stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -806,7 +806,7 @@ function Dense{T}(A::StridedVecOrMat) where T<:VTypes
 end
 function Dense{T}(A::Union{Adjoint{<:Any, <:StridedVecOrMat}, Transpose{<:Any, <:StridedVecOrMat}}) where T<:VTypes
     d = allocate_dense(size(A, 1), size(A, 2), size(A, 1), T)
-    s = unsafe_load(pointer(d))
+    GC.@preserve d s = unsafe_load(pointer(d))
```

Comment:
You need to preserve d "until the function returns." since you are still using the pointer associated with `d`.
The function above also seems to be missing it.